modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -0,0 +1,122 @@
+* You have a scheduler binary.
++
+[NOTE]
+====
+Information on how to create a scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation. Note that the actual functionality of your custom scheduler is not supported by Red Hat.
+====
+* You have created an image containing the scheduler binary and pushed it to a registry.
+
+.Procedure
+
+. Create a file that contains the deployment resources for the custom scheduler:
++
+.Example `custom-scheduler.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: custom-scheduler
+  namespace: kube-system <1>
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: custom-scheduler-as-kube-scheduler
+subjects:
+- kind: ServiceAccount
+  name: custom-scheduler
+  namespace: kube-system <1>
+roleRef:
+  kind: ClusterRole
+  name: system:kube-scheduler
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: custom-scheduler
+  namespace: kube-system <1>
+spec:
+  selector:
+    matchLabels:
+      component: scheduler
+      tier: control-plane
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      serviceAccountName: custom-scheduler
+      containers:
+      - command:
+        - /usr/local/bin/kube-scheduler
+        - --address=0.0.0.0
+        - --leader-elect=false
+        - --scheduler-name=custom-scheduler <2>
+        image: "<namespace>/<image_name>:<tag>" <3>
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+----
+<1> This procedure uses the `kube-system` namespace, but you can use the namespace of your choosing.
+<2> The command for your custom scheduler might require different arguments. For example, you can pass configuration as a mounted volume using the `--config` argument.
+<3> Specify the container image that you created for the custom scheduler.
+
+. Create the deployment resources in the cluster:
++
+[source,terminal]
+----
+$ oc create -f custom-scheduler.yaml
+----
+
+.Verification
+
+* Verify that the scheduler pod is running:
++
+[source,terminal]
+----
+$ oc get pods -n kube-system
+----
++
+The custom scheduler pod should be listed as `Running`:
```

Comment:
There are a couple of instances where "should be" is used. I've seen general advice in reviews to avoid phrases like, "should be", "might be", and so on. 

Suggested edit: "The custom scheduler pod is listed as `Running`:"

This is entirely optional. :)